samples/files/config.sample

```diff
@@ -58,3 +58,21 @@
 #       and /dev/video456 can't be exported, set:
 #       blacklist = video123,video456
 #       Default: <empty>
+#
+# Section [uri]:
+#
+# Keys:
+#   videox
+#       Absolute path to the location where the camera defenition file for videox is hosted
+#       
+# Section [imgcap]
+#    
+# Keys:
+#   location
+#       The local path where captured image will be stored.
```

Comment:
> The local path where captured image will be stored.

The local path with write permission where captured images will be saved. For e.g :  /tmp/